accounts/admin.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin as BaseUserAdmin
+from django.contrib.auth.models import User
+from accounts.models import UserAccount
+
+
+admin.site.register(UserAccount)
+
+
+class AccountInline(admin.StackedInline):
```

Comment:
Why all those classes in the admin.py file? they need to be in the models.py
@ifireball Am I wrong?